packages/formatting/src/utils/devcontainer.ts

```
@@ -0,0 +1,58 @@
+
+/**
+ * Determine if formatters need Node.js
+ */
+export function needsNodeJS(formatters: FormatterDetection): boolean {
+  return !!(formatters.prettier || formatters.remark);
+}
+
+/**
+ * Get recommended VS Code extensions for formatters
+ */
+export function getExtensionsForFormatters(formatters: FormatterDetection): string[] {
```

ESLint

[error] Array type using 'string[]' is forbidden. Use 'Array<string>' instead.

(@typescript-eslint/array-type)